project/src/views/new-evaluation/technicalLevelPicker.js

```diff
@@ -0,0 +1,39 @@
+const TechnicalLevelPickerHeader= function(options = {}) {
+    const headers = options.map(function(h) {        
+      return `<th class="tbl-head tbl-head-skin">${h}</th>`;
+    });
+    const concatHeaders = headers.join("");
```

Comment:
Minor: superfluous variable, can be removed (i.e. its contents `return`ed directly).

Reply:
problem solved